content/change/teaching/resources.md

```diff
@@ -1,13 +1,16 @@
-title: Teaching resources
+title: Resources
```

Comment:
This doesn't have an effect for me (see also https://deploy-preview-20--oskb.netlify.com/change/teaching/resources/) -- I think it's overridden by the overall page structure set in `mkdocs.yml`. I would suggest removing it, as it's no longer necessary.